ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,105 @@
+|------------------|---------------------------------------------------------------------------------------------------------------------------------|
+| kind/enhancement | Enhancement are not bugs or new features but can drastically improve usability or performance of a project component.           |
+| kind/cleanup     | Refactoring code or otherwise clarifying documentation.                                                                         |
+| kind/content     | Content that is not documentation such as help or error messages.                                                               |
+| kind/graphics    | Work involving graphics skill                                                                                                   |
+| kind/regression  | Regressions are usually easy fixes as hopefully the action worked previously and git history can be used to propose a solution. |
+| kind/bug         | Bugs are bugs. The cause may or may not be known at triage time so debugging should be taken account into the time estimate.    |
+| kind/feature     | Functionality or other elements that the project does not currently support.  Features are new and shinny.                      |
+| kind/question    | Contains a user or contributor question requiring a response.                                                                   |
+| kind/usecase     | A description of a user or contributor situation requiring a response perhaps in code or documentation.                         |
+| kind/writing     | Writing documentation, man pages, articles, blogs, or other significant word-driven task.                                       |
+| kind/test        | Tests or test infrastructure needs adding or updating.                                                                                                 |
+
+
+Contributors can add labels by using a `+kind/bug` in an issue or pull request comment.  
+
+### Step 4: Estimate the experience level required
+
+Experience level is a way for a contributor to find an issue based on their
+skill set.  Experience types are applied to the issue or pull request using
+labels.
+
+| Level            | Experience level guideline                                                                                               |
+|------------------|--------------------------------------------------------------------------------------------------------------------------|
+| exp/beginner     | You have made less than 10 contributions in your life time to any open source project.                                   |
+| exp/novice       | You have made more than 10 contributions to an open source project or at least 5 contributions to Docker.                | 
+| exp/proficient   | You have made more than 5 contributions to Docker which amount to at least 200 code lines or 1000 documentation lines.   | 
```

Comment:
I was thinking about that, but not really sure !!  I can do that in v2 if everyone agrees for the change.
Another item is the classification. I see priority/PX as well, so an explanation would be helpful imo.